tests/integration/Core/Framework/Adapter/Twig/fixtures/return-node/array-case.html.twig

```diff
@@ -1,12 +1,11 @@
 {% apply spaceless %}
-    {% macro foo() %}
-        {% return [1, 2] %}
+    {% macro foo(index) %}
```

Comment:
sorry, but we cannot change the test cases. that would mean breaking changes for all templates. 

You need to find a way to change or custom Twig stuff to output what is actually expected

Reply:
Will do, need to dive deeper into TWIG changes :saluting_face: